_posts/2024-05-01-develop-guideline-multiple-data-source-in-opensearch-and-plugins.md

```diff
@@ -0,0 +1,336 @@
+---
+layout: post
+title: "Developer guide: Integrating multiple data sources using OpenSearch Dashboards and plugins"
+authors:
+ - zhyuanqi
```

Comment:
there doesn't seem to be a profile created for 'zhyuanqi' - can you resolve this? 

(would be an appropriate `zhyuanqi.md` file under `_community_members`)

Reply:
Also, got the following error when I ran this locally:

Liquid syntax error (line 245): Expected end_of_string but found colon in "{{ data: indices ? indices : 'Unknown' }}" in /_posts/2024-05-01-develop-guideline-multiple-data-source-in-opensearch-and-plugins.md

Reply:
Hi Kris. I have added the author part/ For `Liquid syntax error (line 245): Expected end_of_string but found colon in "{{ data: indices ? indices : 'Unknown' }}" in /_posts/2024-05-01-develop-guideline-multiple-data-source-in-opensearch-and-plugins.md`, the `{ data: indices ? indices : 'Unknown' }}` is in part of the code block.

Reply:
added